setup-machine.sh

```diff
@@ -201,6 +206,8 @@ sudo ln -s /vx/code/config/grub.cfg /vx/vendor/config/grub.cfg
 # https://up-shop.org/up-bios-splash-service.html
 if [[ "${CHOICE}" == "mark-scan" ]]; then
   sudo cp config/logo-vertical.bmp /vx/code/config/logo.bmp
+elif [[ "${CHOICE}" == "mark" ]]; then
+  sudo cp config/logo-horizontal-800x600.bmp /vx/code/config/logo.bmp
```

Comment:
I think I made a mistake here and this might actually need be the `logo-vertical` to match VxMarkScan